Kickstarter-iOS/Views/Controllers/RewardsCollectionViewController.swift

```diff
@@ -232,6 +232,16 @@ extension RewardsCollectionViewController {
   ) {
     if let rewardCell = cell as? RewardCell {
       rewardCell.delegate = self
+      rewardCell.addCollectionViewController(to: self)
+    }
+  }
+
+  override func collectionView(
+    _: UICollectionView, didEndDisplaying cell: UICollectionViewCell, forItemAt _: IndexPath
+  ) {
+    if let rewardCell = cell as? RewardCell {
+      rewardCell.delegate = nil
+      rewardCell.removeCollectionViewController()
```

Comment:
Can this action not be performed in the `RewardCell`'s `prepareForReuse` function?

Reply:
It could, but considering the addition above, pairing these was more explicit of what's happening...it should also definitely happen in `prepareForReuse` so I'll also add it there.